### PR TITLE
(dbt) Migration enriched_booking_data

### PR DESCRIPTION
**Pull request by carolinemestre-passculture**
# DE PR

## Describe your changes

The purpose of this PR is to migrate the `enriched_booking_data` model, the new name of the model will be `mrt_global__booking` and both tables must be identical.

We are also migrating/updating all the necessary models for the construction of this model.

This PR [adds/removes/fixes/replaces] the [feature/bug/etc]. 

Tag a reviewer if necessacy  @github/username 

## Jira ticket number and/or notion link

JIRA-ticket_number

### Type of change
- [ ] Fix (non-breaking change which corrects expected behavior)
- [ ] New fields (non-breaking change)
- [x] New table (non-breaking change)
- [ ] Concept change (potentially breaking change which modifies fields according to new or evolving business concepts) 
- [ ] Table deletion (potentially breaking change which adds functionality/ table)
      
### Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] My code passes CI/CD tests
- [x] I updated README.md
- [ ] I have updated the dag
- [ ] If my changes concern incremental table, I have altered their schema to accomodate with field's creation/deletion
- [ ] I have made corresponding changes to the [tables documentation](https://www.notion.so/passcultureapp/Documentation-Tables-175a397a8e854ff4a55ae4f3620dbe3b)
- [ ] I have made corresponding changes to the [fields glossary](https://www.notion.so/passcultureapp/854a436a8f1541e1b6ec2a65f8bab600?v=798024ba90404b139e5a17407a3bc604)
- [ ] I will create a review on slack and ensure to specify the duration of the review task: short (<10min), medium (<30min), long (>30min)

### Added tests?
- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help
- [ ] ⏰ no, but I created a ticket

### CODE REVIEW

File: orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__offer.sql [hunk 1]
```diff
@@ -20,6 +20,32 @@ WITH stocks_grouped_by_offers AS (
...
+            ) THEN CONCAT('isbn-',isbn)
+        WHEN (
+            offer_subcategoryId = 'SEANCE_CINE'
+            AND theater_movie_id IS NOT NULL
+            AND theater_movie_id <> ''
+            ) THEN CONCAT('movie_id-',theater_movie_id)
+        ELSE CONCAT('product-',offer_product_id) END AS item_id
+    FROM {{ ref('int_applicative__extract_offer') }}
+    {% if is_incremental() %}
+    AND offer_date_updated BETWEEN date_sub(DATE("{{ ds() }}"), INTERVAL 3 DAY) and DATE("{{ ds() }}")
```
[the remark this answers is not in the document]
Comment (carolinemestre-passculture) [reply orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__offer.sql:42]: Oui, bien vu !

File: orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__deposit.sql [hunk 2]
```diff
@@ -0,0 +1,15 @@
+SELECT
+    id,
+    -- HOTFIX: Adjust 'amount' from 90 to 80 to correct a discrepancy (55 deposit are concerned)
```
[the remark this answers is not in the document]
Comment (carolinemestre-passculture) [reply orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__deposit.sql:3]: ce Hotfix on l'a fait sur l'ancienne stack (donc ça ne concerne pas cette PR). @Jules-Arbelot a créé un ticket et suit ça de près :)

File: orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__offerer.sql [hunk 3]
```diff
@@ -1,10 +1,6 @@
 {% set target_name = target.name %}
 {% set target_schema = generate_schema_name('analytics_dbt_' ~ target_name) %}
 
-{{ config(
```
[the remark this answers is not in the document]
Comment (carolinemestre-passculture) [reply orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__offerer.sql:4]: Si, on doit le rajouter finalement

File: orchestration/dags/data_gcp_dbt/models/mart/global/mrt_global__booking.sql [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    s.offer_subcategoryId,
+    s.physical_goods,
+    s.digital_goods,
+    s.event,
+    s.offer_category_id,
+    ub.user_department_code,
+    ub.user_creation_date,
+    ub.user_activity,
+    s.item_id,
+    rank() OVER (
```
[the remark this answers is not in the document]
Comment (carolinemestre-passculture) [reply orchestration/dags/data_gcp_dbt/models/mart/global/mrt_global__booking.sql:47]: On peut, mais comme on en avait discuté je veux être alignée avec les modèles en production actuellement pour faciliter les data checks.

File: orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__user.sql [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    CASE
+        WHEN user_activity in ("Alternant", "Apprenti", "Volontaire") THEN "Apprenti, Alternant, Volontaire en service civique rémunéré"
+        WHEN user_activity in ("Inactif") THEN "Inactif (ni en emploi ni au chômage), En incapacité de travailler"
+        WHEN user_activity in ("Étudiant") THEN "Etudiant"
+        WHEN user_activity in ("Chômeur", "En recherche d'emploi ou chômeur") THEN "Chômeur, En recherche d'emploi"
+        ELSE user_activity
+    END AS user_activity,
+    CASE
+        WHEN user_civility in ("M", "M.") THEN "M."
+        WHEN user_civility IN ("Mme") THEN "Mme"
```
[the remark this answers is not in the document]
Comment (carolinemestre-passculture) [reply orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__user.sql:30]: Je change en Mme. yes ! Sinon je garde la même logique du `CASE WHEN` pour être alignée avec ce que vous aviez avant.

**Review by LucileRainteau — APPROVED** [review]

**Pull request merged**